docs/README.md

```diff
@@ -159,14 +173,73 @@ This adds a Vector layer to the OpenLayers Map:
 
     addRegions(data) {
 
-        // Regions constructor creates ol.Features from JSON data
-        this.regions_ = new Regions(this, {data: data});
-        this.viewer_.addLayer(new Vector({source : this.regions_}));
+        if (this.regions_ instanceof Regions) {
+            this.regions_.initialize_(this.regions_, data);
+        } else {
+            var options = {};
+            if (data) options['data'] = data;
+            // Regions constructor creates ol.Features from JSON data
+            this.regions_ = new Regions(this, options);
+        }
 
         // enable roi selection by default, as well as modify and translate
         this.regions_.setModes([SELECT, MODIFY, TRANSLATE]);
     }
 
+We now have 2 representations of the ROIs: ``model.regions_info.data`` Map of Roi
+objects containing Shapes, and the Open Layer Features on the source Regions layer
+of the viewer. These are kept in-sync via events posted on the event bus, such as
+selection, deletion, editing and visibility (e.g. Z/T index change).
+
+Pagination of ROIs & Shapes
+---------------------------
+
+If the number of ROIs on the image is greater than ``ROI_PAGE_SIZE`` (default setting is 500)
+then we only load ROIs that have a Shape on the current Z/T plane.
+ROIs will also be paginated within each plane if over 500 ROIs per plane.
+ROIs will only have loaded the Shapes on the current plane. Other Shapes in the ROI won't
+be loaded, but the shape_count is still displayed on the `regions-list` table.
+When a user expends the ROI in the table (or a Shape is selected by clicking on the image or
```

Comment:
```suggestion
When a user expands the ROI in the table (or a Shape is selected by clicking on the image or
```

Reply:
Maybe the observed syntax is intentional, but it does not match the other "observed" occurences in this doc.